test/integration/policy_report_metric_test.go

```diff
@@ -48,9 +48,30 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test policyreport_info metric", fu
 		By("Setting the insights client to poll every minute")
 		insightsClient, err := common.OcHub("get", "deployments", "-n", ocmNS, "-l", insightsClientSelector, "-o", "name")
 		Expect(err).To(BeNil())
+		insightsClientPod, err := common.OcHub("get", "pods", "-n", ocmNS, "-l", insightsClientSelector, "-o", "name")
+		Expect(err).To(BeNil())
+		insightsClientPod = strings.Split(insightsClientPod, "pod/")[1]
 		insightsClient = strings.TrimSpace(insightsClient)
 		_, err = common.OcHub("set", "env", "-n", ocmNS, insightsClient, "POLL_INTERVAL=1")
 		Expect(err).To(BeNil())
+		//checking if new pod has spun up
+		Eventually(func() interface{} {
+			var err error
+			pod, err := common.OcHub("get", "pods", "-n", ocmNS, "-l", insightsClientSelector, "--field-selector=status.phase=Running,metadata.name!="+insightsClientPod)
```

Comment:
This command is complex, but as we discussed I don't think we have a better way. And it's consistent with the second check, so that's good.